### ISSUE TITLE
Notification cannot be dismissed

### ISSUE THREAD

**Issue by Issung** [jellyfin/jellyfin-android#378]
Watched a movie last night casting to my Chromecast, once it's finished I stop the play, close the app and the Chromecast is now free, but the notification that says the movie is still playing remains, and cannot be dismissed unless I restart the phone.

Most apps with notifications like this would have an "x" button somewhere to dismiss the notification.

![image](https://user-images.githubusercontent.com/22212465/88471066-c9978700-cf47-11ea-9f21-5007d7c32899.png)

**Comment by CutterXYZ** [comment]
I agree, it's quite annoying. Until this is fixed, as a workaround you can connect to the server and "stop" playback from the phone.
